Client/Frontend/Browser/Onboarding/OnboardingShieldsView.swift

```diff
@@ -0,0 +1,118 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+import BraveShared
+
+extension OnboardingShieldsViewController {
+    
+    private struct UX {
+        /// A negative spacing is needed to make rounded corners for details view visible.
+        static let negativeSpacing: CGFloat = -16
+        static let descriptionContentInset: CGFloat = 32
+    }
+    
+    class View: UIView {
+        
+        let continueButton = CommonViews.primaryButton().then {
+            $0.accessibilityIdentifier = "OnboardingShieldsViewController.ContinueButton"
```

Comment:
Really nice job adding these.

Reply:
I haven't tested it but at minimum we should allow user to dismiss the onboarding flow so I added it